low-coverage/readme.md

```diff
@@ -0,0 +1,16 @@
+# Remove Low Coverage Fields
+
+This script removes all the fields that have coverage less than a
+given threshold coverage. For example, if given a dataset of 1000 rows
+and a threshold value of 0.10, all fields with more than 900 missing
+values will be eliminated.
+
+#Inputs
+
+-  The dataset you wish to alter
+
+- A threshold value between 0 and 1
```

Comment:
i think perhaps it'd be nice to make this parameter also accept an integer number.  if someone passes a value in [0,1), it's interpreted as a ratio, just as you do know, and if one passes an integer >= 1, it's taken as an absolute number of rows (say, i want to keep those with less than 23 missings)... 0 has the same meaning as a ratio and as a count, and 1 as a ratio is not really useful, is it?